_posts/2016-09-06-bazel-rules-protobuf.md

```diff
@@ -995,6 +994,8 @@ via the
 and
 [grpc_gateway_binary](https://github.com/pubref/rules_protobuf/tree/master/grpc_gateway#grpc_gateway_binary) rules, so you can easily bridge your gRPC apps with HTTP/1.1 gateways.
 
+Refer to the [complete list of supported languages and gRPC versions](https://github.com/pubref/rules_protobuf/#rules) for more information.
```

Comment:
When 1.0.0 came out, I tried to upgrade but was blocked by issues noted in https://github.com/pubref/rules_protobuf/issues/9.  I'll try again with a later commit.

As for java, I just have not done it yet, but I'm pretty confident about upgrading it.  I'll go ahead try that now and merge if it works out.